src/main/java/teammates/ui/webapi/action/ActionFactory.java

```diff
@@ -98,8 +99,7 @@
         map(ResourceURIs.STUDENT_EDIT_DETAILS, GET, GetStudentEditDetailsAction.class);
         map(ResourceURIs.COURSE_STUDENT_DETAILS_EDIT, PUT, PutCourseStudentDetailsEditAction.class);
         map(ResourceURIs.COURSE_EDIT_DETAILS, GET, GetCourseEditDetailsAction.class);
-        map(ResourceURIs.COURSE_EDIT_DETAILS_SAVE, PUT, SaveCourseEditDetailsAction.class);
-        map(ResourceURIs.COURSE_DELETE, DELETE, DeleteCourseAction.class);
+        map(ResourceURIs.COURSE_DELETE, DELETE, SoftDeleteCourseAction.class);
```

Comment:
Why there is a duplicated entry above `map(ResourceURIs.BIN_COURSE, PUT, SoftDeleteCourseAction.class);`?